src/rockstor/system/services.py

```diff
@@ -33,6 +33,7 @@
 SUPERCTL_BIN = ('%s/bin/supervisorctl' % settings.ROOT_DIR)
 SUPERVISORD_CONF = ('%s/etc/supervisord.conf' % settings.ROOT_DIR)
 NET = '/usr/bin/net'
+WBINFO = '/usr/bin/wbinfo'
```

Comment:
just to double check, you've installed samba-winbind-clients package for /usr/bin/wbinfo, correct?